app/models/event.rb

```diff
@@ -12,6 +12,8 @@
 class Event < ActiveRecord::Base
   translates :title, :introduction, :conclusion
   belongs_to :location
+  # suggestion from Nicholas
+  to_param :title
```

Comment:
As a reference for my future self: http://apidock.com/rails/ActiveRecord/Integration/ClassMethods/to_param